### PR TITLE
[WIP/NI] Announcements highlight in the Homepage

### PR DESCRIPTION
**Pull request by Espina2**
screenshot_20161125_194005

So after I talk with @jospoortvliet  about how we can highlight some things that are new I came out with this solution. 

Since @jancborchardt don't like carousels (and we have some valid points) I think its the best way to do it. 

@nextcloud/designers

### CODE REVIEW

File: page-homepage.php [hunk 1]
```diff
@@ -6,6 +6,10 @@
 <!-- 	<div class="container-fluid"> -->
 		<div class="row">
 			<div class="col-md-6 toptext ">
+				<a class="annoucement" href="#">
+				<span class="type">Fresh</span>
+				<span class="message"><strong>Nextcloud 11 has just arrive.</strong> Take a look at the new features</span>
```
Comment (tcitworld) [inline page-homepage.php]: arrived

Comment (Espina2) [inline page-homepage.php]: The text is dummy, so it doesn't matter at this point. :)

**Comment by jospoortvliet** [comment]
I like it! Absolutely works for me. Really cool, actually. If Jan agrees, we should merge it then just hide it until we need it.

**Comment by Espina2** [comment]
Yes you can comment the code and use it only when it needed. But for example you can already highlight the beta version. :)

**Comment by jancborchardt** [comment]
@Espina2 just one thing: Can you put it _below_ the »Get Nextcloud« button? Nothing should ever be above the slogan and call to action.

**Comment by jancborchardt** [comment]
(Btw the animated logo looks cool as well! :) cc @jospoortvliet all my mail apps have issues these days for some reason, so here’s the reply ;)

**Comment by Espina2** [comment]
I attach the image like you want. I didn't like it, it makes this not work good in mobile devices, and dont look anymore like an highlight. 

screenshot_20161130_115900
screenshot_20161130_115934

**Comment by jancborchardt** [comment]
@Espina2 to fix the layout issue on mobile, put the »Fresh« tag in a separate line above maybe :)

> I didn't get it why it should be not above the slogan

Because it’s information overload. It almost looks like a banner ad, which people are prone to ignore. Nothing should take attention away from the slogan, the short description and the call to action. That’s the most important thing on the page.

**Comment by Espina2** [comment]
> Because it’s information overload. It almost looks like a banner ad, which people are prone to ignore. 

Pushing the thing above doesn't change this. 

I don't agree with you, having this on top just look something new are come, this is the purpose of having this, doesn't make any sense if this are the last thing you are reading. I prefer to have on top or not have it all. 

Since moving this above is not my vision and doesn't represent what I think is best, feel free to change yourself or erase this PR.

**Comment by jospoortvliet** [comment]
Honestly I agree with @Espina2 here, if only because it simply looks better above than below... And while I get that it might distract from the main message, well, it is news, fresher... I think it is fine this way and could be merged. Jan, can you agree to disagree?

**Comment by jancborchardt** [comment]
Ok whatever, go ahead.

cc @nextcloud/designers what do you think?

**Comment by Espina2** [comment]
Let me just give a quick adjustment in mobile.

**Comment by jospoortvliet** [comment]
@Espina2 you can merge once OK.

**Comment by Espina2** [comment]
Merged

**Comment by Espina2** [comment]
@jospoortvliet don't forget to change the link and the text. :)

**Pull request merged**